javascripts/downloads.js

```diff
@@ -55,9 +60,25 @@ function onLoad() {
         oStableVersionElement.append(sStableRelease);
 
         //replace spaceholder with values from JSON
+	if (bBeta === true) {    
```

Comment:
you can merge the two if's by putting the unstableRelease stuff into the second if

Reply:
Ok, please re-check if I did it correctly.